dspace/config/ldn/announce-relationship

```diff
@@ -13,11 +13,12 @@
 ##            {10} uri of the related resource
 ##            {11} relationship type, i.e. http://purl.org/vocab/frbr/core#supplement
 ##            {12} urn:uuid of the relation (randomly generated)
+##            {14} config 'dspace.ui.url'
```

Comment:
check comment below

Reply:
The same as above: I will amend the description of the templates to reflect that param 0 could be: config 'dspace.ui.url' or submitter email id (e.g. 'mailto:submitter-email') depending on the service configuration

Because all templates share parameters I had to add a separate one to always be 'dspace.ui.url'